src/main/java/internet/shop/controller/AddToBucketController.java

```diff
@@ -0,0 +1,29 @@
+package internet.shop.controller;
+
+import internet.shop.lib.Inject;
+import internet.shop.service.BucketService;
+import internet.shop.service.UserService;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AddToBucketController extends HttpServlet {
+    private static final Long DEFAULT_ID = 0L;
```

Comment:
```suggestion
    private static final Long DEFAULT_USER_ID = 0L; 
``` 
if I understand correctly?